build/ci-build.yml

```
@@ -74,6 +74,15 @@ stages:
             inputs:
               artifact: 'Build'
               path: '$(Build.SourcesDirectory)'
+          - task: AzureCLI@2
+            displayName: 'Get Token for Azure Management API'
+            inputs:
+              azureSubscription: XXXX
+              scriptType: pscore
+              scriptLocation: inlineScript
+              inlineScript: |              
+                $token= & az account get-access-token --resource="https://management.azure.com/" --query accessToken
+                Write-Host "##vso[task.setvariable variable=AzureManagementApiAccessToken]$token"
```

Maybe align with other naming:
```suggestion
                Write-Host "##vso[task.setvariable variable=Azure.ManagementApi.AccessToken]$token"
```